### DISCUSSION TITLE
Navigation region change from JP to ROW - MHI2_JP_VW48x_P0829

### DISCUSSION THREAD

**Discussion by baazjones** [Mr-MIBonk/M.I.B._More-Incredible-Bash#560] [General]
Greetings, noob and new owner of a Japan import Golf 7.5 here, Discover Pro unit. 

I checked the documentation but could only find region conversions to all other regions except ROW when looking at the MIB menu layout, is this possible to convert from JP to ROW? I am in Africa so ROW will need to be installed. If this is not possible at all, is there a method of having the carplay maps display in the digital cluster, currently it is trying to load the built in maps only which is not working due to the location of the car. 

What other codings will need to be done if this is possible please? 

Thank you in advance.

**Comment by baazjones** [comment]
I have managed to do this, Thanks https://github.com/Mr-MIBonk for this wonderful M.I.B, without it this would have NOT been possible.

**Comment by rizkhan7** [comment]
Could you please list the steps you took???

**Comment by baazjones** [comment]
What is the part number of your radio?

**Comment by rizkhan7** [comment]
Part No : 3G0035050D

Train Info : MHI2_JP_VW48x_P0651
MU Version : 0637

Variant Info : FM2-H-N-JP-VW-MQB
Diagnosis Variant : G2_HIGH_MMX_NAV_JP_VW
Main Unit Type : FM2 - Front MU MIB2

Installed MIB version is v3.6.0

Downloaded 2025 Row full from here : https://mib-helper.com/index.php?train=MHI2_JP_VW48X_P0651#details

Have also applied the aio patch for my firmware...

Did do a region change, but it took really long to get back into service menu and android auto so panicked and rolled region back to JP and now all is good...

**Reply by m9638ymct0718-collab** [reply]
Hi rizkhan7,

I have a MHI2_JP_VW48x_P0636 / MU0636 unit and I’m trying to find the P0651 / MU0637 full firmware.

You mentioned that you did a region change and then “rolled region back to JP”.

Did you actually reinstall the original P0651 / MU0637 JP firmware when you returned to JP, or did you only change the region setting back to JP?

If you still have the original P0651 / MU0637 full firmware, could you please tell me where you got it from?

I’m specifically looking for the full JP firmware, not the M.I.B. patch.

Thank you!

**Comment by baazjones** [comment]
My method seems to be confirmed on radios with the following part number: 5NA035050*

I am not 100% sure if this will work on your part number too, or if it is prone to brick the unit. If your purpose is only to have built in navigation and you would not want to run the risk of bricking your unit, I would suggest using CP or AA rather. For radio frequencies, it can be changed using VCDS or OBD11. 

Should you wish to continue with the conversion, I have done a conversion from JP to ER (ER = Europe and RoW) using Mr Bonk's MIB. GEM -> region conversion. After that, you will first need to install the ER firmware, check compatibility on mib-helper.com and download firmware from mibsolution.one. You will need to add the FEC code for your region as well for maps to work. Once completed, you will be able to update maps via long press menu and then update, standard update from navigation button will not work for the conversion. If FEC is not added, maps will come up for around 2 mins and will then come up with an error "Nav data is not valid".

Hope this helps.

Afterword: It seems that the conversion of radios is kept high secret, I joined a few facebook pages and asked for help, I immediately got DM's asking for 100 Euros to do the conversion, residing in Africa this amounts to quite a fee after forex. After I converted the radio, I posted on those pages that I managed (without instructions) and got blocked on those pages... Why can't it be like Mr Bonk's software? FREE.

**Comment by knox2** [comment]
Hi @baazjones, i have the 5NA035050B, whats the process of conversion from Japan to Africa, tried to read what you had typed above and really didn't understand it well

**Reply by baazjones** [reply]
Hi there @knox2, for me it was not understandable in the beginning as well, I had to do A LOT of googling and reading before I could figure it out. Please let me know where you are stuck, I will provide more guidance on what I did to complete the conversion

**Comment by owehussain** [comment]
Hi @baazjones hope your well, I needed to change my radio on my Volkswagen  from Japan to eu, can you please tell me the process I don’t mind paying you, thank you

**Reply by baazjones** [reply]
Hi, what radio model do you have? This was only tested on the Discover Pro MIB2.5 unit with part number 5NA035050*. I have some instructions on this feed, but just be aware that you can brick your radio.

**Comment by i1d9** [comment]
@baazjones 
<img width="3024" height="4032" alt="IMG_236C724F-CB63-4B2F-896E-B73B86C5574E" src="https://github.com/user-attachments/assets/4fab032e-e0e3-41ce-acb5-54071543ad2c" />

I'm also in Africa, do you mind guiding me, I have a 2018 mk7.5

**Reply by baazjones** [reply]
Perfect, this is the exact one that I have, and converted it to ROW, running 2026 maps.

**Reply by i1d9** [reply]
nice, what East African countries does RoW cover? kindly share the steps did to install the maps and the link to the 2026 maps

**Reply by baazjones** [reply]
<img width="673" height="671" alt="image" src="https://github.com/user-attachments/assets/618fb366-c243-4aa4-b8be-8997433fef22" />

This is the map for ROW. In my previous comment, I mentioned the process, you first need to install MrBonk, then do firmware update to EU, then do navigation region conversion in GEM, then format nav db, then update maps from service menu update. Again, make sure ignition is on, car not running, and have a charger on your battery, this process can easily take up to 3 hours, and you do not want the car to shut off during the process.
